v2.0/select-clause.md

```diff
@@ -0,0 +1,451 @@
+---
+title: Simple SELECT Clause
+summary: The Simple SELECT clause loads or computes data from various sources.
+toc: false
+---
+
+The simple `SELECT` clause is the main SQL syntax to read and process
+existing data.
+
+When used as a standalone statement, the simple `SELECT` clause is
+also called "the `SELECT` statement". However, it is also a
+[selection clause](selection-clauses.html) that can be combined
+with other constructs to form more complex [relational
+expressions](relational-expressions.html).
+
+<div id="toc"></div>
+
+## Synopsis
+
+{% include sql/{{ page.version.version }}/diagrams/simple_select_clause.html %}
+
+<div markdown="1"></div>
+
+{{site.data.alerts.callout_success}}The simple <code>SELECT</code> clause also has other applications not covered here, such as executing <a href="functions-and-operators.html">functions</a> like <code>SELECT current_timestamp();</code>.{{site.data.alerts.end}}
+
+## Required Privileges
+
+The user must have the `SELECT` [privilege](privileges.html) on the tables used as operands.
+
+## Parameters
+
+| Parameter | Description |
+|-----------|-------------|
+| `DISTINCT` or `ALL` | See [Eliminate Duplicate Rows](#eliminate-duplicate-rows). |
+| `DISTINCT ON ( a_expr [, ...] )` | `DISTINCT ON` followed by a list of [scalar expressions](scalar-expressions.html) within parentheses. See [Eliminate Duplicate Rows].
+| `target_elem` | A [scalar expression](scalar-expressions.html) to use to compute a column in each result row, or `*` to automatically retrieve all columns from the `FROM` clause.<br><br>If `target_elem` contains an [aggregate function](functions-and-operators.html#aggregate-functions), a `GROUP BY` clause can be used to further control the aggregation. |
+| `table_ref` | The [table expression](table-expressions.html) you want to retrieve data from.|
+| `AS OF SYSTEM TIME timestamp` | Retrieve data as it existed as of [`timestamp`](timestamp.html).<br/><br/>For more information, see [this example](#select-historical-data-time-travel) or our blog post [Time-Travel Queries](https://www.cockroachlabs.com/blog/time-travel-queries-select-witty_subtitle-the_future/).<br/><br/>Because `AS OF SYSTEM TIME` returns historical data, your reads might be stale. |
+| `WHERE a_expr` | Only retrieve rows that return `TRUE` for `a_expr`, which must be a [scalar expression](scalar-expressions.html) that returns Boolean values using columns (e.g., `<column> = <value>`).  |
+| `GROUP BY expr_list` | When using [aggregate functions](functions-and-operators.html#aggregate-functions) in `target_elem` or `HAVING`, list the column groupings in `expr_list`. |
```

Comment:
In the diagram, it looks like we inlined `expr_list`, so we should either adjust this row or the diagram.